index.html

```diff
@@ -167,6 +167,22 @@ <h4>LET'S CONNECT</h4>
    </div>
    </section>
    </section>
+<section class="contactPage">
+<div class="container">
+    <h2 class="contactMe">Contact me</h2>
+    <p class="descriptionContact">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+<form>
+    <input type="text" name="name" id="name" placeholder="Type your name" required minlength="2" maxlength="30" value="Yeremias">
+    <br>
+    <input type="email" name="email" id="email" placeholder="Type your email" required value="natajayanj14@gmail.com">
+    <br>
+    <textarea placeholder="Write your message here" required maxlength="500" minlength="2"></textarea>
+    <br/>
+    <button class="button">Get It Touch</button>
+</form>
```

Comment:
- According to the project requirements, your form should use the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html).